Automattic/wp-calypso
# Update dispatchPostRevisionsDiffsRequest to use dispatchRequestEx

#### Changes proposed in this Pull Request

Part of the ongoing transition from `dispatchRequest` to `dispatchRequestEx`

#### Testing instructions

1. Go to posts, edit a post, and click History
2. Make sure revisions are visible
3. Retry with a post with 1 and many revisions

#25121
